app/views/reservations/_reservation_fields.html.haml

```diff
@@ -1,8 +1,20 @@
+- start_editable = !f.object.persisted? || f.object.reserve_start_at_editable?
 .datetime-block
-  = f.input :reserve_start_date, :input_html => { :class => 'datepicker' }
+  - if start_editable
+    = f.input :reserve_start_date, :input_html => { :class => 'datepicker' }
+  - else
+    .control-group
+      %span
+        %b Reserve Start
+      .controls
+        = f.object.reserve_start_date
   .control-group
     .controls
       .string.optional.control-label &nbsp;
-      = time_select f, :reserve_start, minute_step: @instrument.reserve_interval
+      - if start_editable
+        = time_select f, :reserve_start, minute_step: @instrument.reserve_interval
+      - else
+        .controls
```

Comment:
I'm not sure, I've been experimenting with the styles because I can't get it to vertically align how i want. I may need some help with this too.